### ISSUE TITLE
1.8.3 breaks gulp-ng-templates

### ISSUE THREAD

**Issue by RobeeeJay** [tracker1/gulp-header#43]
I'm not sure what has changed between 1.8.2 and 1.8.3 but something is causing gulp-ng-templates to break now with the following error:

```
fs.js:839
  return binding.lstat(pathModule._makeLong(path));
                 ^

Error: ENOENT: no such file or directory, lstat '/home/project-barbie/build/source/js/main/directives/partials/templates.min.js'
    at Error (native)
    at Object.fs.lstatSync (fs.js:839:18)
    at DestroyableTransform.TransformStream [as _transform] (/home/project-barbie/node_modules/gulp-ng-templates/node_modules/gulp-header/index.js:38:12)
    at DestroyableTransform.Transform._read (/home/project-barbie/node_modules/gulp-ng-templates/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:159:10)
    at DestroyableTransform.Transform._write (/home/project-barbie/node_modules/gulp-ng-templates/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:147:83)
    at doWrite (/home/project-barbie/node_modules/gulp-ng-templates/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:313:64)
    at writeOrBuffer (/home/project-barbie/node_modules/gulp-ng-templates/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:302:5)
    at DestroyableTransform.Writable.write (/home/project-barbie/node_modules/gulp-ng-templates/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:241:11)
    at write (/home/project-barbie/node_modules/gulp-concat/node_modules/readable-stream/lib/_stream_readable.js:623:24)
    at flow (/home/project-barbie/node_modules/gulp-concat/node_modules/readable-stream/lib/_stream_readable.js:632:7)
```

This may either be the way gulp-ng-templates is using it, or it could be a wider issue that affects other dependencies and uses.

**Comment by RIAstar** [comment]
See #36

**Comment by tracker1** [comment]
try 1.8.5, will reopen if it's still an issue.

**Comment by tracker1** [comment]
1.8.6

**Comment by tracker1** [comment]
see #44

**Comment by RobeeeJay** [comment]
It's fixed in 1.8.6, thanks!
